draft-irtf-cfrg-vdaf.md

```diff
@@ -1680,41 +1684,41 @@ def run_vdaf(
-times on the same sequence of reports.
+Depending on the VDAF, verification, aggregation, and collection may be carried
+out multiple times on the same sequence of reports.
 
 In practice, VDAF execution is distributed across Clients, Aggregators, and
-Collectors that exchange messages (i.e., report shares, prep shares, and
+Collectors that exchange messages (i.e., report shares, verifier shares, and
 aggregate shares) over an insecure network. The application must therefore take
 some additional steps in order to securely execute the VDAF in this
 environment. See {{security}} for details.
 
-## Communication Patterns for Preparation {#vdaf-prep-comm}
+## Communication Patterns for Verification {#vdaf-verify-comm}
 
-The only stage of VDAF execution that requires interaction is preparation
-({{sec-vdaf-prepare}}). There are a number of ways to coordinate this
+The only stage of VDAF execution that requires interaction is verification
+({{sec-vdaf-verify}}). There are a number of ways to coordinate this
 interaction; the best strategy depends largely on the number of Aggregators
 (i.e., `vdaf.SHARES`). This section describes two strategies, one specialized
 for two Aggregators and another that is suitable for any number of Aggregators.
 
-In each round of preparation, each Aggregator writes a prep share to some
-broadcast channel, which is then processed into the prep message using the
-public `prep_shares_to_prep()` algorithm and broadcast to the Aggregators to
-start the next round. The goal of this section is to realize this broadcast
-channel.
+In each round of verification, each Aggregator writes a verifier share to some
+broadcast channel, which is then processed into the verifier message using the
+public `verifier_shares_to_message()` algorithm and broadcast to the
+Aggregators to start the next round. The goal of this section is to realize
+this broadcast channel.
 
 The state machine of each Aggregator is shown below.
 
 ~~~ aasvg
- +--> Rejected <--+   +----------------+   Finished(out_share)
- |                |   |                |           ^
- |                |   |                v           |
-Start ----> Continued(prep_state, prep_round, outbound)
- |                                                 |
- |                                                 v
- +-----------------> FinishedWithOutbound(out_share, outbound)
+ +--> Rejected <--+   +--------------+   Finished(out_share)
+ |                |   |              |           ^
+ |                |   |              v           |
+Start -----> Continued(verify_state, verify_round, outbound)
+ |                                               |
+ |                                               v
+ +---------------> FinishedWithOutbound(out_share, outbound)
 ~~~
-{: #vdaf-prep-state-machine title="State machine of VDAF preparation."}
+{: #vdaf-ver-state-machine title="State machine of VDAF verification."}
```

Comment:
While this is just a slug, it's probably one of the few remaining instances of "ver", so we may as well change it.
```suggestion
{: #vdaf-verification-state-machine title="State machine of VDAF verification."}
```